### PR TITLE
release: 2.51.3

### PR DESCRIPTION
**Pull request by anonymouse64**
Update changelog with the 2.51.3 changes.

### CODE REVIEW

File: packaging/debian-sid/changelog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+snapd (2.51.3-1) unstable; urgency=medium
+
+  * New upstream release, LP: #1929842
+    - interfaces/builtin: add sd-control interface (#10522)
```
Comment (mvo5) [inline packaging/debian-sid/changelog:4]: (nitpick^2) I would just remove the "(#10522) here as we generally don't include the PR number.

**Review by mvo5 — COMMENTED** [review]
Thanks, looks fine! just one nitpick

**Review by pedronis — APPROVED** [review]
thanks

File: packaging/debian-sid/changelog (hunk 1 above)
Comment (anonymouse64) [reply packaging/debian-sid/changelog:4]: ack

**Pull request merged**